src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -695,11 +702,10 @@ protected ArrayBuilder<Symbol> AddOrWrapTupleMembers(ImmutableArray<Symbol> curr
                                                                                         tupleFieldIndex,
                                                                                         locations,
                                                                                         isImplicitlyDeclared: defaultImplicitlyDeclared);
+                                        members.Add(defaultTupleField);
                                     }
                                 }
 
-                                members.Add(defaultTupleField);
```

Comment:
>members.Add(defaultTupleField); [](http://example.com/codeflow?start=32&length=31)

The refactoring doesn't look equivalent.

Reply:
That's intentional. We don't add any new members in the `IsDefinition` case on line 685. The purpose of this method changed, it is now adding new members, rather that rebuilding the complete list.